Live-Snap/Live-Snap/Controllers/ExportWallpaperViewController.swift

```diff
@@ -158,23 +160,28 @@ class ExportWallpaperViewController: UIViewController, PHLivePhotoViewDelegate {
         System.shared.appDelegate().pageViewController?.setViewControllers([cropWallpaperViewController], direction: .reverse, animated: true, completion: nil)
     }
     
+    func showSaveSuccessPopUpView() {
+        view.isUserInteractionEnabled = false
+        view.addSubview(saveSuccessBackgroundView)
+        view.addSubview(saveSuccessPopUpView)
+        
+        UIView.animate(withDuration: 0.25, delay: 0.0, options: .curveEaseOut, animations: {
+            self.saveSuccessPopUpView.center.y = self.view.center.y
+            self.saveSuccessBackgroundView.alpha = 0.75
+        }) { (finished: Bool) in
+            if finished {
```

Comment:
this will always succeed so no need for the if finished, you dont handle the failure case anyway so this won't work if finished = false